core/helpers/preamble.h

```diff
@@ -82,9 +82,9 @@ struct L3CNTLRegisterOffset {
 
 namespace DebugModeRegisterOffset {
 template <typename GfxFamily>
-constexpr uint32_t registerOffset = 0x20ec;
+static constexpr uint32_t registerOffset = 0x20ec;
```

Comment:
Only reg_configs.h change is necessary to unbreak build. This one is for consistency.